fdw/pxf_filter.c

```diff
@@ -192,65 +212,221 @@ dbop_pxfop_map pxf_supported_opr_op_expr[] =
 	{1755 /* numericle */ , PXFOP_LE},
 	{1757 /* numericge */ , PXFOP_GE},
 	{1753 /* numericne */ , PXFOP_NE},
+
+	/* generic array comparison operators */
+	{ARRAY_EQ_OP /* array_eq */, PXFOP_EQ},
+	{1071 /*array_ne */, PXFOP_NE},
+
+	/* bytea */
+	{ByteaEqualOperator /* byteaeq */ , PXFOP_EQ},
+	{1957 /* bytealt */ , PXFOP_LT},
+	{1959 /* byteagt */ , PXFOP_GT},
+	{1958 /* byteale */ , PXFOP_LE},
+	{1960 /* byteage */ , PXFOP_GE},
+	{1956 /* byteane */ , PXFOP_NE},
+	{2016 /* bytealike */ , PXFOP_LIKE},
+	{2017 /* byteanlike */ , PXFOP_NOTLIKE},
+
+	/* time */
+	{TimeEqualOperator /* time_eq */ , PXFOP_EQ},
+	{1110 /* time_lt */ , PXFOP_LT},
+	{1112 /* time_gt */ , PXFOP_GT},
+	{1111 /* time_le */ , PXFOP_LE},
+	{1113 /* time_ge */ , PXFOP_GE},
+	{1109 /* time_ne */ , PXFOP_NE},
+
+	/* timestamp with time zone */
+	{TimestampTZEqualOperator /* timestamptz_eq */ , PXFOP_EQ},
+	{1322 /* timestamptz_lt */ , PXFOP_LT},
+	{1324 /* timestamptz_gt */ , PXFOP_GT},
+	{1323 /* timestamptz_le */ , PXFOP_LE},
+	{1325 /* timestamptz_ge */ , PXFOP_GE},
+	{1321 /* timestamptz_ne */ , PXFOP_NE},
+
+	/* interval */
+	{IntervalEqualOperator /* interval_eq */ , PXFOP_EQ},
+	{1332 /* interval_lt */ , PXFOP_LT},
+	{1334 /* interval_gt */ , PXFOP_GT},
+	{1333 /* interval_le */ , PXFOP_LE},
+	{1335 /* interval_ge */ , PXFOP_GE},
+	{1331 /* interval_ne */ , PXFOP_NE},
+
```

Comment:
```suggestion

	/* uuid */
```
(like in external table)